state-sync/state-sync-v1/tests/test_harness.rs

```diff
@@ -897,7 +897,7 @@ impl ExecutorProxyTrait for MockExecutorProxy {
         Ok(0)
     }
 
-    fn publish_on_chain_config_updates(
+    fn publish_events_and_reconfig_notifications(
```

Comment:
`publish_event_notifications` might just be enough, but I'll leave it to you

Reply:
Done, thanks!